src/main/spawn-background-process.ts

```diff
@@ -8,11 +10,15 @@ async function spawnBackgroundProcess() {
   const childProcess = await import('node:child_process')
   const path = await import('node:path')
   const { fileURLToPath } = await import('node:url')
-
   const { createLogFilePath } = await import('./core/log/create-log-file-path.js')
-
   const date = new Date().toISOString()
   const logFilePath = createLogFilePath(date)
+  const environment = new Environment()
+
+  if (environment.isCI === false || environment.isTelemetryEnabled === false) {
+    // Do nothing and exit quietly
+    return
+  }
```

Comment:
does this superseed https://github.com/ibm-telemetry/telemetry-js/blob/main/src/main/ibm-telemetry.ts#L65-L74? if so, should we remove?